src/components/Navbar.vue

```diff
@@ -17,96 +29,133 @@ export default {
       <v-btn to="/" flat>Home</v-btn>
       <v-btn to="/dashboard" flat>Dashboard</v-btn>
       <v-btn to="/leaderboard" flat>Leaderboard</v-btn>
-      <v-btn class="login-btn" icon="mdi-account-circle" size="small" flat></v-btn>
+
+      <v-menu transition="slide-y-transition" v-if="currentUser">
+        <template v-slot:activator="{ props }">
+          <v-btn
+            class="login-btn"
+            icon="mdi-account-circle"
+            size="small"
+            flat
+            v-bind="props"
+          ></v-btn>
+        </template>
+        <v-list>
+          <v-list-item>
+            <v-list-item-title class="d.flex justify-space-between mb-2">
+              <i
+                aria-hidden="true"
+                class="v-icon notranslate mdi mdi-account-circle mr-4"
+              ></i>
+              <span>{{ currentUser }}</span>
+            </v-list-item-title>
+            <v-list-item-title class="d.flex justify-space-between logout">
+              <span @click="LogOut">
```

Comment:
would it be helpful if we made this an anchor tag that links to the "/api/logout" route like it's done with the "Log in" v-btn?